core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -475,14 +506,6 @@ public ContractEvent getEvent(String eventName) {
         return this.getEvents().get(eventName);
     }
 
-    /**
-     * Getter function for caver.
-     * @return Caver
-     */
-    public Caver getCaver() {
```

Comment:
If you remove this function, the code that uses `contract.getCaver()` cause error.